### DISCUSSION TITLE
Unable to view all guides on wiki

### DISCUSSION THREAD

**Discussion by rabwill** [dotnet/dev-proxy#1235] [Q&A] [answered]
I am currently [exploring the tool](https://github.com/microsoftgraph/msgraph-developer-proxy/wiki/How-to-guides) and is unable to go to wiki under [Simulate behaviours](https://github.com/microsoftgraph/msgraph-developer-proxy/wiki/How-to-guides#simulate-behaviours). It says `You do not have permission to update this wiki.`

**Comment by waldekmastykarz** [comment]
Have you signed in using SSO with your corp account? Since this repo is under Microsoft org, and your account is also tied to Microsoft, you likely need to sign in using SSO first to access any resources in this repo.

**Comment by rabwill** [comment]
Yes @waldekmastykarz I am signed in using SSO for corp account. I tried again, it's still the same issue. 

![wiki-issue](https://user-images.githubusercontent.com/20379311/212014884-6dc37436-b269-41ec-9c8b-fdef6b5efa33.gif)

**Comment by garrytrinder** [comment, marked as answer]
Got it. We had a broken link.

I've updated the link, so should be working for you now @rabwill

**Comment by waldekmastykarz** [comment]
Can we close this issue as fixed @garrytrinder?
